packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -31,24 +35,77 @@ const propTypes = {
 const defaultProps = {
   disableUnselectedRows: false,
   onChange: undefined,
+  onBlur: undefined,
   selectedIndexes: [],
 };
 
+const rowsSelector = 'row-group';
+
 class SelectableTableRows extends React.Component {
   constructor(props) {
     super(props);
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleOnBlur = this.handleOnBlur.bind(this);
+  }
+
+
+  componentDidMount() {
+    // Adds ARIA-Label for Pre-selected rows
+    const rowGroup = document.querySelector(`#${rowsSelector}`);
```

Comment:
This will query the entire dom, if there are multiple tables this logic will affect all of them. It will also run for every table on the page each time a new table mounts. 

We should investigate a solution that adds the aria labels before the table is rendered into the dom.

Reply:
Fix is suppose to add aria-label for all the tables in page which has pre-selected rows. I agree that it will run for every table on page each time a new table mounts but that is how it is supposed to be since it should add aria-label to every table on page having pre-selected row within it. 

As you mentioned in other comment this can be done at [clonedChildItems](https://github.com/cerner/terra-core/blob/f4e469fc37e008087364c2d01dfcabee2fe1b83f/packages/terra-table/src/SelectableTableRows.jsx#L169) to add aria attribute before tables are mounted. 

I think this can be done by adding aria attribute as prop to react component. but I would like to know if possible how this can be done or if there are any better way to do this.